### DISCUSSION TITLE
How does this project compare to Apache Fury?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [eclipse-serializer/serializer#144] [General]
https://fury.apache.org/

**Comment by fh-ms** [comment]
Eclipse Serializer is optimized for Java only. Core benefits are security and extensibility, achieved by all aspects mentioned on the [main page](https://github.com/eclipse-serializer/serializer).

At first glance, Apache Fury has different goals, mainly speed and multi-language support. 

They also want to stay compatible with the built-in Java Serialization. I don't know if they can eliminate the shortcomings and vulnerabilities of the Java Serialization by doing so, but I highly doubt it.
